### DISCUSSION TITLE
DRM Removal for NOOK

### DISCUSSION THREAD

**Discussion by Prysmarine** [noDRM/DeDRM_tools#348] [Q&A] [answered]
I have set DeDRM to get the key from the NOOK windows app, and can import a book that was downloaded on my tablet and open it in Calibre. But it only shows the cover and then goes to a contents page. Nothing else is contained.

**Comment by ElleKayEm** [comment, marked as answer]
Is it a comic?  Those epubs have a non-standard structure which their own app can handle but calibre's ebook viewer cannot.  A workaround is to take out the full page images and zip them up to make cbz format (which you could convert if desired).  Regardless sounds like the DRM is removed.

**Reply by Prysmarine** [reply]
It is a manga so I would assume it falls under the same category as far as that goes. How would I go about taking out the images?

**Reply by ElleKayEm** [reply]
An epub is also a zip file.  Some zip utilities will unzip them directly.  Or you can change the extension from epub to zip and unzip it.

**Reply by Prysmarine** [reply]
Ok, I see the pages and stuff. I'll have to play around with it a lot more and figure out what will work where. Thank you!
